apps/admin/frontend/src/components/import_cvrfiles_modal.tsx

```diff
@@ -252,11 +252,11 @@ export function ImportCvrFilesModal({ onClose }: Props): JSX.Element | null {
   if (currentState.state === 'duplicate') {
     return (
       <Modal
-        title="Duplicate File"
+        title="Duplicate Export"
         content={
           <P>
-            The selected file was ignored as a duplicate of a previously loaded
-            file.
+            The selected export was ignored as a duplicate of a previously
+            loaded export.
```

Comment:
Because exports are now (and have for a while been) directories instead of single files, I've moved away from "file" as the descriptor for a CVR export and shifted to "export"